hack/prune-api-description-whitelist.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Is it wrong to have? I'm going to be writing these things and it's pretty useful I will admit ...